crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -105,15 +139,37 @@ pub struct StandardMaterial {
 
     /// Support two-sided lighting by automatically flipping the normals for "back" faces
     /// within the PBR lighting shader.
-    /// Defaults to false.
+    ///
+    /// Defaults to `false`.
     /// This does not automatically configure backface culling, which can be done via
     /// `cull_mode`.
     pub double_sided: bool,
-    /// Whether to cull the "front", "back" or neither side of a mesh
-    /// defaults to `Face::Back`
+
+    /// Whether to cull the "front", "back" or neither side of a mesh.
+    ///
+    /// Defaults to `Face::Back`.
     pub cull_mode: Option<Face>,
+
+    /// Whether to apply only the base colour to this material.
+    ///
+    /// Normals, occlusion textures, roughness, metallic, reflectance and
+    /// emissive are ignored if this is set to `true`.
     pub unlit: bool,
+
+    /// How to interpret the alpha channel of the `base_color_texture`.
+    ///
+    /// By default, it's `Opaque`, therefore completely ignored.
+    /// Note that currently bevy handles poorly semi-transparent textures. You
+    /// are likely to encounter the following bugs:
+    ///
+    /// - When two `AlphaMode::Blend` materials occupy the same pixel, only one
+    ///   material's color will show.
+    /// - If a different mesh is both "in front" and "behind" a non-opaque material,
+    ///   bevy won't know which material to display in front, which might result in
+    ///   flickering.
```

Comment:
I think the way the `AlphaMode` type works should probably be documented on that type instead.
```suggestion
    /// How to apply the alpha channel of the `base_color_texture`. See [`AlphaMode`] for details.
    /// Defaults to [`AlphaMode::Opaque`].
```

And the documentation for AlphaMode could be:
```rust
#[derive(Component, Debug, Default, Reflect, Copy, Clone, PartialEq)]
#[reflect(Component, Default)]
pub enum AlphaMode {
    /// Base color alpha values are overridden to be fully opaque (1.0).
    #[default]
    Opaque,
    /// An alpha cutoff must be supplied where alpha values >= the cutoff
    /// will be fully opaque (1.0) and < will be fully transparent (0.0).
    Mask(f32),
    /// The base color alpha value defines the opacity of the color.
    /// Standard alpha-blending is used to blend the fragment's color with the color behind it.
    Blend,
}
```

Reply:
I think documentation of workarounds for shortcomings of the currently implementation or so should be elsewhere such as in the bevy book. What do you think @alice-i-cecile ?

Reply:
I feel pretty strongly that this should live with the code. It's not helpful for beginners, and experts encountering weird bugs will dive into the API docs in detail.

Reply:
Definitely agree that these docs belong on AlphaMode though!